### DISCUSSION TITLE
Get count only?

### DISCUSSION THREAD

**Discussion by gibb3h** [kamranayub/igdb-dotnet#21] [Q&A] [answered]
Hi, the api allows for a call to https://api.igdb.com/v4/games/count with a query to get a count of records, I intended to use this to help with pagination, is there any way to accomplish this, as I don't see a separate end point in IGDBClient.Endpoints and setting the endpoint manually ("games/count") encodes the forward slash to %2F and results in a 404

Cheers
Gib

**Comment by kamranayub** [comment]
Currently there isn't, I can convert this to an issue to address

**Comment by kamranayub** [comment, marked as answer]
This was added in the last release using the `IGDBClient.CountAsync` method:

```csharp
var res = await _api.CountAsync(IGDBClient.Endpoints.Games, "where id = 4;");
res.Count;
```
